### PR TITLE
add option to show number on the left

### PR DESCRIPTION
**Pull request by amosbird**
https://github.com/company-mode/company-mode/issues/889

### CODE REVIEW

**Comment by dgutov** [comment]
Hi! Thanks for the patch.

Two notes:

I think we can get by with just one new value of `company-show-numbers` instead of a new variable (the value would be symbol `left`).

This feature needs at least one test.

**Comment by amosbird** [comment]
ok, updated

File: NEWS.md [hunk 1]
```diff
@@ -13,6 +13,7 @@
   map binding `TAB` to "jump to the next field"). Affects both
   `company-template` and `yasnippet` (requires changes from 2019-04-21,
   currently unreleased).
+* New values for option `company-show-numbers` to show number on the left.
```
Comment (dgutov) [inline NEWS.md]: "New value"

File: company.el [hunk 2]
```diff
@@ -641,7 +641,9 @@ commands in the `company-' namespace, abort completion."
 (defcustom company-show-numbers nil
   "If enabled, show quick-access numbers for the first ten candidates."
   :type '(choice (const :tag "off" nil)
-                 (const :tag "on" t)))
+                 (const :tag "left" 'left)
+                 (const :tag "right" 'right)
```
Comment (dgutov) [inline company.el]: Why `right`? Just use `t` for this.

Comment (amosbird) [reply company.el]: It's for some user who imagines symmetric configurations... I'll remove it

**Comment by dgutov** [comment]
OK, let's try it. Thank you!

**Pull request merged**